retriever/engines/mysql.py

```diff
@@ -122,6 +120,11 @@ def get_connection(self):
 
         args['client_flag'] = client.LOCAL_FILES
         self.get_input()
-        return dbapi.connect(charset=self.lookup_encoding(),
-                             read_default_file='~/.my.cnf',
-                             **args)
+        try:
+            return dbapi.connect(charset=self.lookup_encoding(),
+                             read_default_file='~/.my.cnf', **args)
+        except dbapi.err.OperationalError as _:
```

Comment:
I think there is no need of `as _` in the above statement. @henrykironde do we also want to print the actual error?

Reply:
@apoorvaeternity @henrykironde  Should I remove the first exception?